woocommerce/woocommerce
# Make "Proceed to Checkout" button non-sticky when zoom level is bigger than 100%

### Changes proposed in this Pull Request:





When the screen width is smaller than 782px, the `Proceed to Checkout` button becomes sticky. When a user is zooming into the screen, the `Proceed to Checkout` button could overlap the product items. This PR prevents that the `Proceed to Checkout` button becomes sticky when the screen height is smaller than 500px, to ensure that the products items remain visible even with a high zoom factor.

Closes #43637.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add a product to the cart.
2. Zoom into the page to decrease the screen height.
3. Verify that the `Proceed to order` button is sticky when the screen height is bigger than 500px, and the screen width is smaller than 782px.
4. Verify that the `Proceed to order` button is no longer sticky when the screen height is 500px or less, and the screen width is smaller than 782px.

### Screenshots

<table>
<tr>
<td valign="top">
Zoom: 100% / Width: 600px / Height: 1000px
<br><br>
<img width="598" alt="Screenshot 2024-06-20 at 15 17 16" src="https://github.com/woocommerce/woocommerce/assets/3323310/48082102-663f-406d-9554-4ca2ac28f7dd">
</td>
<td valign="top">
Zoom: 200% / Width: 300px / Height: 500px
<br><br>
<img width="592" alt="Screenshot 2024-06-20 at 15 18 24" src="https://github.com/woocommerce/woocommerce/assets/3323310/fb6b99b8-67e3-4d4f-a86a-44289a023629">
</td>
</tr>
</table>



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
